Docker/start-container.sh

```diff
@@ -3,14 +3,14 @@
 #Set user and group
 umask 0002
 PUID=${PUID:-`id -u squeezeboxserver`}
-PGID=${PGID:-`id -g squeezeboxserver`}
+PGID=${PGID:-`id -g users`}
```

Comment:
Hmm... I believe I changed that because I got permissions issue with `user`, or "group not found" or similar. I'll have to double check.

Reply:
"user" is not a group. You need to set it to "users".

Reply:
That was a typo in my comment. As you can see from my below log snippet I put `users` there.

BTW: I'm seeing the same complaint on Synology, too. Would we have to change the `Dockerfile`, too?

Reply:
Try now. I've changed to nogroup. Maybe for some reason, users group does not exist with you.